microsoft/onefuzz
# assign scaleset to a role

## Summary of the Pull Request

Allows the nodes in the scaleset to access the service by assigning their managed identity to the ManagedNode Role

## PR Checklist


## Info on Pull Request

_What does this include?_

## Validation Steps Performed

- Enable mandatory role assignment in your deployment
- Create a scaleset
- Execute the following  `python src\deployment\registration.py assign_scaleset_role <onefuzz_instance_name> <scaleset_name>
- Run a job on the and observe that it runs successfully
